thirdeye/thirdeye-frontend/tests/integration/pods/components/filter-bar/component-test.js

```diff
@@ -1,16 +1,30 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
+import config from 'thirdeye-frontend/mocks/filterBarConfig';
 
 moduleForComponent('filter-bar', 'Integration | Component | filter bar', {
   integration: true
 });
 
-test('it renders', function(assert) {
+test('it renders well', function(assert) {
```

Comment:
@ttbach not using arrow functions here because the `this` context changes otherwise.